src/app/test/test.component.html

```diff
@@ -35,98 +35,13 @@
 
     <p>Generator is {{ generatorEnabled|booleanToString: 'enabled':'disabled' }}</p>
 
-    <div class="table-responsive">
-      <table class="table table-hover">
-        <thead>
-        <tr>
-          <th>
-            <input data-cy-test="toggleSelectAll" type="checkbox" [checked]="amountOfSelectedReports === reports?.length && amountOfSelectedReports > 0"
-                   (click)="toggleSelectAll()">
-          </th>
-          @if (showStorageIds) {
-            <th>StorageId</th>
-          }
-          <th>(Re)run/open</th>
-          <th>Name</th>
-          <th>Description</th>
-          <th>Variables</th>
-          <th>Run result</th>
-          <th>Options</th>
-        </tr>
-        </thead>
-        <tbody data-cy-test="table">
-          @for (report of reports; track report.storageId) {
-
-            @if (matches(report)) {
-              <tr>
-                <td>
-                  <input data-cy-test="reportChecked" type="checkbox" [checked]="report.checked"
-                         (click)="toggleCheck(report)">
-                </td>
-                @if (showStorageIds) {
-                  <td>
-                    {{ report.storageId }}
-                  </td>
-                }
-                <td>
-                    <span class="buttons">
-                      <button
-                        data-cy-test="runReport"
-                        title="Run report"
-                        class="btn btn-info button-width"
-                        (click)="run(report)"
-                      ><i class="fa fa-play"></i></button>
-                      <button
-                        data-cy-test="openReport"
-                        title="Open report"
-                        class="btn btn-info button-width"
-                        (click)="openReport(report.storageId)"
-                      ><i class="fa fa-folder-open"></i></button>
-                    </span>
-                </td>
-                <td>
-                  {{ getFullPath(report.path, report.name) }}
-                </td>
-                <td>
-                  {{ report.description }}
-                </td>
-                <td>
-                  @if (report.variables) {
-                    <span data-cy-test="variables">{{ extractVariables(report.variables) }}</span>
-                  }
-                </td>
-                <td>
-                  @if (report.reranReport) {
-                    <span data-cy-test="runResult"
-                          [style.color]="report.reranReport.color">
-                      {{ report.reranReport.resultString }}
-                    </span>
-                  }
-                  @if (report.error) {
-                    <span style="color: red">
-                      {{ report.error }}
-                    </span>
-                  }
```

Comment:
In the new test table component, you haven't add this to the table

Reply:
Added it back again. Also noticed there was a missing disabled attribute for the replaceReport button which is now also added.